### DISCUSSION TITLE
Redirect to the Custom OIDC provider login page

### DISCUSSION THREAD

**Discussion by Preeyank** [nextauthjs/next-auth#11963] [Help]
Question:  How can i redirect my user directly to the **custom provider login page**?
Currently I have this path "api/auth/signin" (default provided by Auth.Js)  which the user will be redirected to if the user is not Authenticated. This workflow is working fine.  
How to remove this added signin button click on "api/auth/signin" and instead just go directly the custom provider login page

![Screenshot 2024-10-03 at 11 47 48 AM](https://github.com/user-attachments/assets/5b6ec8b4-d920-4863-a116-8f89d85b571f)

Attached the screenshot. This page is what I want to avoid and go directly to the custom provider page which the user will be redirected to once clicked on the sign in button.

**Comment by andyburgessmd** [comment]
I'd also really love to know how to do this!
